pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -1,4 +1,4 @@
-yum -y update
+yum -y -x WALinuxAgent update
```

Comment:
There's a yum update later in the script as well (after repositories are added), should we exclude there as well?

Reply:
I only see two yum installs in the remainder of the files

Reply:
I don't see the other update there is only an install of a specific rpm [here](https://github.com/Azure/ARO-RP/pull/2512/files#diff-a7a118c3717408a3425f9209b28c891edad1c68d9b9a67cb41b18545f8ce37e3R16) and then a list of specific packages on [here](https://github.com/Azure/ARO-RP/pull/2512/files#diff-a7a118c3717408a3425f9209b28c891edad1c68d9b9a67cb41b18545f8ce37e3R74) am I missing it?

Reply:
You're right, they are not updates. LGTM!